pkg/kyverno/apply/command.go

```diff
@@ -127,7 +128,22 @@ func Command() *cobra.Command {
 				}
 			}
 
-			if len(resourcePaths) == 0 && !cluster {
+			var resources []*unstructured.Unstructured
+			if resourcePaths[0] == "-" {
+				if isInputFromPipe() {
+					resourceStr := ""
+					scanner := bufio.NewScanner(os.Stdin)
+					for scanner.Scan() {
+						resourceStr = resourceStr + scanner.Text() + "\n"
+					}
+
+					yamlBytes := []byte(resourceStr)
+					resources, err = getResource(yamlBytes)
+					if err != nil {
+						fmt.Println("error: ", err)
```

Comment:
should we return an error back to the user from here instead of continuing?

Reply:
Yes, we should return an error back to user. I have updated the code.